File: src/0204_statistical_inference/00_live_class/script_05.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
"""
Created on Sat Nov 11 09:45:16 2023

@author: jerry
"""


# %%

import pandas as pd

from scipy import stats



# %%

data = pd.read_csv("./data/0204_statistical_inference/01_introduction_and_parametric_tests/Correlation test.csv")
data.info()



# %%

stats.pearsonr(data['aptitude'], data['job_prof'])

File: src/0204_statistical_inference/00_live_class/script_05.R

library(nortest)


data <- read.csv("./data/0204_statistical_inference/01_introduction_and_parametric_tests/Correlation\ test.csv", header = TRUE)

head(data)
summary(data)


qqnorm(data$aptitude, col = "cadetblue")
shapiro.test(data$aptitude)
lillie.test(data$aptitude)

qqnorm(data$job_prof, col = "cadetblue")
shapiro.test(data$job_prof)
lillie.test(data$job_prof)


cor.test(data$aptitude, data$job_prof, alternative = 'two.sided', var.equal = TRUE)

